astropy/convolution/tests/test_kernel_class.py

```diff
@@ -201,7 +201,7 @@ def test_multiply_scalar_type(self, number):
         """
         gauss = Gaussian1DKernel(3)
         gauss_new = number * gauss
-        assert type(gauss_new) == Gaussian1DKernel
+        assert isinstance(gauss_new, Gaussian1DKernel)
```

Comment:
I think you need to keep the test here: we don't want the multiplication to suddenly start returning subclasses!  However, it should be
```
assert type(gauss_new) is Gaussian1DKernel
```